spec/neworder_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'neworder'
+require 'menu'
+require 'timestamp'
+require 'sms'
+
+describe NewOrder do
+  let(:neworder) { described_class.new }
```

Comment:
We might usually phrase this like so:

```
subject(:new_order) { described_class.new }
```

See this:

> A named subject improves on the explicit subject by assigning it a contextually
semantic name. Since a named subject is an explicit subject, it still defines the value that is returned by the subject method in the example scope. However, it defines an additional helper method with the provided name. This helper method is memoized. The value is cached across multiple calls in the same example but not across examples.
https://www.relishapp.com/rspec/rspec-core/v/3-5/docs/subject/explicit-subject

By convention we'd make the name `new_order` rather than `NewOrder` — but hard to know that unless you see code that does it I suppose!